instrumentation/src/javatests/org/oppia/android/instrumentation/player/ExplorationPlayerTest.kt

```diff
@@ -78,60 +87,77 @@ class ExplorationPlayerTest {
     device.findObjectByText("SUBMIT")?.click()
     device.findObjectByText("CONTINUE")?.click()
-
-    // Multiple Choice Interaction - 1.
+  }
+  /** Play first MultipleChoiceInteraction for PrototypeExporation. */
+  private fun playMultipleChoiceIntearction1() {
     device.findObjectByText("Eagle")?.click()
     device.findObjectByText("CONTINUE")?.click()
-
-    // Multiple Choice Interaction - 2.
+  }
+  /** Play second MultipleChoiceInteraction for PrototypeExporation. */
+  private fun playMultipleChoiceIntearction2() {
     device.findObjectByText("Green")?.click()
     device.findObjectByText("CONTINUE")?.click()
+  }
 
-    // Item Selection Interaction.
+  /** Play second ItemSelectionInteraction for PrototypeExporation. */
+  private fun playItemSelectionInteraction() {
     device.findObjectByText("Red")?.click()
     device.findObjectByText("Green")?.click()
     device.findObjectByText("Blue")?.click()
     device.findObjectByText("SUBMIT")?.click()
     device.findObjectByText("CONTINUE")?.click()
+  }
 
-    // Numeric Input Interaction.
+  /** Play second NumericInputInteraction for PrototypeExporation. */
+  private fun playNumericInputInteraction() {
     device.findObjectByRes("numeric_input_interaction_view")?.text = "121"
     device.findObjectByText("SUBMIT")?.click()
     device.findObjectByText("CONTINUE")?.click()
+  }
 
-    // Ratio Input Interaction.
+  /** Play second RatioInputInteraction for PrototypeExporation. */
+  private fun playRatioInputInteraction() {
     device.findObjectByRes("ratio_input_interaction_view")?.text = "4:5"
     device.findObjectByText("SUBMIT")?.click()
     device.findObjectByText("CONTINUE")?.click()
+  }
 
-    // Text Input Interaction.
+  /** Play second TextInputInteraction for PrototypeExporation. */
+  private fun playTextInputInteraction() {
     device.findObjectByRes("text_input_interaction_view")?.text = "Finnish"
     device.findObjectByText("SUBMIT")?.click()
     device.findObjectByText("CONTINUE")?.click()
+  }
 
-    // Drag And Drop Interaction.
+  /** Play second DragAndDropInteraction for PrototypeExporation. */
+  private fun playDragAndDropInteraction() {
     device.findObjectByDesc("Move item down to 2")?.click()
     device.findObjectByDesc("Move item down to 3")?.click()
     device.findObjectByDesc("Move item down to 4")?.click()
     device.findObjectByText("SUBMIT")?.click()
     device.findObjectByText("CONTINUE")?.click()
+  }
 
-    // Drag Drop Merge Interaction.
+  /** Play second DragDropMergeInteraction for PrototypeExporation. */
+  private fun playDragDropMergeInteraction() {
     device.findObjectByDesc("Link to item 2")?.click()
     device.findObjectByDesc("Move item down to 3")?.click()
     device.findObjectByText("SUBMIT")?.click()
     device.findObjectByText("CONTINUE")?.click()
+  }
 
-    // End Exploration.
+  /** End exploration for PrototypeExploration. */
+  private fun endExploration() {
```

Comment:
Suggest: ``playEndExplorationInteraction`` (technically it's another interaction).

Reply:
Done